classificators/src/test/java/ru/okpdmarket/controller/ClassificatorControllerTest.java

```diff
@@ -85,30 +84,43 @@ public void getClassificatorTypes() throws Exception {
         this.mockMvc.perform(get("").accept(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk())
                 .andDo(document("classificators", preprocessResponse(prettyPrint()), responseFields(
-                        fieldWithPath("[].code").description("English name of classificator type. To use in URL"),
+                        fieldWithPath("[].code").description("Unique identifier of classificator type. To use in URL"),
                         fieldWithPath("[].name").description("Localized name of Classificator type"),
                         fieldWithPath("[].description").description("Description"))));
     }
 
     @Test
     public void getTopItems() throws Exception {
-        this.mockMvc.perform(get("/OKPD").accept(MediaType.APPLICATION_JSON))
+        this.mockMvc.perform(get("/okpd").accept(MediaType.APPLICATION_JSON))
                 .andExpect(status().isOk())
                 .andDo(document("classificator-top-items", preprocessResponse(prettyPrint())));
     }
 
+    /*
```

Comment:
Is it an example of `prettyPrint`? Please add explanation.

Reply:
nope, removed this garbage

Reply:
Strange comment is still here